File: sdks/go/examples/quickstart.go
// Quickstart against a running http-admin (sak330-d).
package main

import (
	"fmt"
	"log"
	"os"

	sak "github.com/tycheung/swissarmynoife-sdk"
)

func main() {
	base := os.Getenv("SAK_HTTP")
	if base == "" {
		base = "http://127.0.0.1:8787"
	}
	c := sak.NewClient(base)
	health, err := c.Health()
	if err != nil {
		log.Fatal(err)
	}
	modules, err := c.ListModules()
	if err != nil {
		log.Fatal(err)
	}
	fmt.Printf("health=%v\nmodules=%v\n", health, modules)
}

File: sdks/java/examples/Quickstart.java
package com.swissarmynoife.sdk;

/** Quickstart against a running http-admin (sak331-d). */
public final class Quickstart {
  public static void main(String[] args) throws Exception {
    String base = System.getenv().getOrDefault("SAK_HTTP", "http://127.0.0.1:8787");
    SakClient c = new SakClient(base);
    System.out.println("health=" + c.health());
    System.out.println("modules=" + c.listModules());
  }
}
